File: iconics-java/src/main/java/com/open/iconics/view/IconicsCheckBox.java
package com.open.iconics.view;

import android.content.Context;
import android.util.AttributeSet;

public class IconicsCheckBox extends IconicsCompoundButton {

    public IconicsCheckBox(Context context) {
        this(context, null);
    }

    public IconicsCheckBox(Context context, AttributeSet attrs) {
        this(context, attrs, android.R.attr.checkboxStyle);
    }

    public IconicsCheckBox(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @Override
    public CharSequence getAccessibilityClassName() {
        return IconicsCheckBox.class.getName();
    }
}

File: iconics-kotlin/src/main/java/com/open/iconics/view/IconicsCheckBox.kt
package com.open.iconics.view

import android.content.Context
import android.util.AttributeSet

open class IconicsCheckBox @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyle: Int = androidx.appcompat.R.attr.checkboxStyle
) : IconicsCompoundButton(context, attrs, defStyle) {

    override fun getAccessibilityClassName(): CharSequence {
        return IconicsCheckBox::class.java.name
    }
}
